### ISSUE TITLE
Paid options ?

### ISSUE THREAD

**Issue by badgone88** [1RandomDev/showly-oss#1]
Hi,

Is there any way to unlock paid options ?

Thanks for this fork btw, very useful without Google trackers ;)

**Comment by 1RandomDev** [comment]
Currently you can just tap the "Single Payment" button 7 times, but for later releases that will probably change and you will only be able to unlock premium by opening the file `/data/data/com.michaldrabik.showly_oss/shared_prefs/PREFERENCES_MISC.xml` in a ROOT explorer and add the line `<boolean name="KEY_PREMIUM" value="true" />` before the last line. Or just downgrade to 3.20.1, unlock and upgrade to the latest version.

**Comment by sombresu** [comment]
> Currently you can just tap the "Single Payment" button 7 times, but for later releases that will probably change and you will only be able to unlock premium by opening the file `/data/data/com.michaldrabik.showly_oss/shared_prefs/PREFERENCES_MISC.xml` in a ROOT explorer and add the line `<boolean name="KEY_PREMIUM" value="true" />` before the last line. Or just downgrade to 3.20.1, unlock and upgrade to the latest version.

Is this still work? I have tried to paste the code on line 11 and it doesn't work.

**Comment by 1RandomDev** [comment]
Yes nothing has changed in the settings format since then. Just make sure the code is inside the `<map> ... </map>` group like the other settings.

**Comment by nightznero** [comment]
Helloz,

I Lil modder here,if You want to unlock premium u can also make changes in _.smali_ files.
To do this find in Dex explorer(any dex editor) path **z5** -> **j** and search for **KEY_PREMIUM** below this here is a code 0x0 that's mean u are free user,change it to 0x1 to unlock it. It should looks like this:
**const/4 p3, 0x0**
and after unlock:
**const/4 p3, 0x1**
Best Regards, NIGHTZNERO.
